### PR TITLE
John wp post

### PR DESCRIPTION
**Pull request by labboy0276**

### CODE REVIEW

File: src/content/articles/wordpress-performance-essentials.md [hunk 1]
```diff
@@ -0,0 +1,440 @@
...
+
+Once you go through and [setup your site with Cloudflare](https://support.cloudflare.com/hc/en-us/articles/201720164-Step-2-Create-a-Cloudflare-account-and-add-a-website) then you are ready to unleash the fury.  *Note: There is a [CloudFlare WordPress plugin](https://wordpress.org/plugins/cloudflare/), but you don't need it if you follow this whole guide.  Most caching plugins come with a CloudFlare (or any CDN) plugin.*
+
+Here are the settings you should have on each of the sections/apps of CloudFlare:
+
+##### **DNS**
+
+  * Click the orange cloud (http proxy) on all A and CNAME records.
+  * Also, you should utilize CNAME flattening as well. To do this create a new CNAME record and put www in the name and then @ in the domain name.
+
```
Comment (serundeputy) [inline src/content/articles/wordpress-performance-essentials.md]: Maybe a screenshot of cloudflare dash and an example string with the `@`?

File: src/content/articles/wordpress-performance-essentials.md [hunk 2]
```diff
@@ -0,0 +1,440 @@
...
+[CloudFlare](https://www.cloudflare.com/) is a no brainer when it comes to quick and easy performance setup.  CloudFlare is a Content Delivery Network service that also takes over the DNS of your site.  So this means you will need access to your site's registrar.  CloudFlare's free plan is more than enough to get a feel for how awesomesauce this service is.
+
+Once you go through and [setup your site with Cloudflare](https://support.cloudflare.com/hc/en-us/articles/201720164-Step-2-Create-a-Cloudflare-account-and-add-a-website) then you are ready to unleash the fury.  *Note: There is a [CloudFlare WordPress plugin](https://wordpress.org/plugins/cloudflare/), but you don't need it if you follow this whole guide.  Most caching plugins come with a CloudFlare (or any CDN) plugin.*
+
+Here are the settings you should have on each of the sections/apps of CloudFlare:
+
+##### **DNS**
+
+  * Click the orange cloud (http proxy) on all A and CNAME records.
+  * Also, you should utilize CNAME flattening as well. To do this create a new CNAME record and put www in the name and then @ in the domain name.
```
Comment (serundeputy) [inline src/content/articles/wordpress-performance-essentials.md]: get rid of `Also` or `as well` (don't need both)

File: src/content/articles/wordpress-performance-essentials.md [hunk 3]
```diff
@@ -0,0 +1,440 @@
...
+##### **Crypto**
+
+  * Dependent on your hosting providers settings, you will chose Flexible or Full in the SSL setting.  Start with Flexible, change all the settings below and then try Full.  If Full causes a 5xx error on your site, change back to Flexible, purge the caches in the Caches app and give it a minute.
+  * Go down to Always use HTTPS and click it to on.  Also head down to Automatic HTTPS Rewrites and click it to on. All of these settings will enable https on your site, for free!  Just go back through your site to make sure you have no assets stuck on http.  You will see the mixed content message in the console.  Usually it is one or two assets, but easy enough to fix real quick.
+
+
+##### **Speed**
+
+  * In the Auto Minify settings, check all 3 boxes for CSS, JS, and HTML.  This will reduce the page size on top of any other plugins you may be using.
+  * You can also try enabling the RocketLoader, but I have always had mix results with it.  Test it out to see if it helps any.
```
Comment (serundeputy) [inline src/content/articles/wordpress-performance-essentials.md]: ~~mix~~ to `mixed`

File: src/content/articles/wordpress-performance-essentials.md [hunk 4]
```diff
@@ -0,0 +1,440 @@
...
+
+
+Nginx
+-----
+
+"Apache is like Microsoft Word, it has a million options but you only need six.  Nginx does those six things, and it does five of them 50 times faster than Apache".
+
+There are numerous articles out there they weigh the advantages of Nginx over Apache.  We could go on and on about the two of them, but if you want performance, go with Nginx. If you are still curious, [here is an article from the Nginx Community Wiki](https://www.nginx.com/resources/wiki/community/why_use_it/) on the advantages of Nginx.
+
+Most hosting environment come with Nginx ready to go typically, like [Pantheon](https://pantheon.io/) and [Platform](https://platform.sh/).  If you are hosting through a do it yourself provider (like [Digital Ocean](https://www.digitalocean.com/)), then here are a couple tips and tricks.
```
Comment (serundeputy) [inline src/content/articles/wordpress-performance-essentials.md]: Most hosting ~~environment~~ `environments` come with Nginx ready to go. For example [Pantheon](https://pantheon.io/) and [Platform](https://platform.sh/) both use Nginx.  If you are hosting through a do it yourself provider (like [Digital Ocean](https://www.digitalocean.com/)), then here are a couple tips and tricks.

File: src/content/articles/wordpress-performance-essentials.md [hunk 5]
```diff
@@ -0,0 +1,440 @@
...
+
+
+##### **Cache All Of Your Static Resources**
+
+Caching static assets like CSS, JS, images, etc has a huge benefit for your site.  You will deliver these assets faster to your end user while reducing the load on the server.  It is a win-win to set this up.  It is pretty simple and easy to do.
+
+Go to /etc/nginx/sites-available folder on your server.  Edit the file which correlates to your site (usually the default file).  You can do this with the command ```sudo nano default``` or whatever CLI editor you use.  Add the following settings to the file:
```
Comment (serundeputy) [inline src/content/articles/wordpress-performance-essentials.md]: ~~correlates~~ to `corresponds`

File: src/content/articles/wordpress-performance-essentials.md [hunk 6]
```diff
@@ -0,0 +1,440 @@
...
+
+##### **CDN Tab**
+  * Enable the CloudFlare settings tab
+
+You can then setup the CloudFlare settings in its respective tab and purge the caches from the WordPress site!  That's it for the settings, really simple yet quite powerful in nature.  Play around with other settings if you want to see what results you may get.
+
+Autoptimize Plugin
+------------------
+
+The [Autotimize](https://wordpress.org/plugins/autoptimize/) plugin is used to aggregate css / js and inline css on your site.  I mainly use this plugin to inline the CSS, but use its optimize options as well.  It is lightweight and very easy to setup.
```
Comment (serundeputy) [inline src/content/articles/wordpress-performance-essentials.md]: ~~ Autotimize~~ to `Autoptimize`

File: src/content/articles/wordpress-performance-essentials.md (hunk 2 above)
Comment (labboy0276) [reply src/content/articles/wordpress-performance-essentials.md]: done

File: src/content/articles/wordpress-performance-essentials.md (hunk 1 above)
Comment (labboy0276) [reply src/content/articles/wordpress-performance-essentials.md]: done

File: src/content/articles/wordpress-performance-essentials.md (hunk 3 above)
Comment (labboy0276) [reply src/content/articles/wordpress-performance-essentials.md]: done

File: src/content/articles/wordpress-performance-essentials.md (hunk 4 above)
Comment (labboy0276) [reply src/content/articles/wordpress-performance-essentials.md]: done

File: src/content/articles/wordpress-performance-essentials.md (hunk 5 above)
Comment (labboy0276) [reply src/content/articles/wordpress-performance-essentials.md]: done

File: src/content/articles/wordpress-performance-essentials.md (hunk 6 above)
Comment (labboy0276) [reply src/content/articles/wordpress-performance-essentials.md]: done

**Comment by pirog** [comment]
:shipit:

**Pull request merged**